src/runtime/entitlements-manager-test.js

```diff
@@ -103,6 +108,15 @@ describes.realWin('EntitlementsManager', {}, (env) => {
     deps = new DepsDef();
     sandbox.stub(deps, 'win').returns(win);
     const globalDoc = new GlobalDoc(win);
+
+    globalDoc.getRootNode = function () {
```

Comment:
Arrow functions could help this code be more concise. Ex:
```

Reply:
Thanks for the suggestion. Simplified accordingly.